specs/sb/api.md

```diff
@@ -342,6 +400,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
```

Comment:
Yes, as we discussed before, schema is defined as JSON schema, which consists of a "properties" section, possibly in addition to "required" and "definitions". As such, each different schema defined in the schemas section must include at least a subfield "properties" to match JSON schema.